faculty_cli/cli.py

```diff
@@ -473,55 +505,70 @@ def server():
     is_flag=True,
     help="Print extra information about servers.",
 )
-def list_servers(project, all, verbose):
-    """List your Faculty servers."""
-    project_id = _resolve_project(project)
-
+def list_servers(provided_project, all, verbose):
+    (
+        """List your Faculty servers."""
+        """\n\nIf you do not specify a project, all servers will be listed."""
+    )
     status_filter = None if all else ServerStatus.RUNNING
-    servers = _get_servers(project_id, status=status_filter)
+    if not provided_project:
+        projects = [(project.id, project.name) for project in _list_projects()]
+        user_id = faculty_cli.auth.user_id()
```

Comment:
Could you instead use this to get the user ID?
```
account_client = faculty.client("account")
user_id = account_client.authenticated_user_id()
```
`faculty_cli.auth.user_id()` is an old part of the CLI library that will get removed in future.

Reply:
Done. I wrapped it in a _get_user_id() helper rather than inline, and replaced other instances of `auth.user_id`